evap/staff/templates/staff_course_email.html

```diff
@@ -2,18 +2,52 @@
 
 {% load i18n %}
 {% load static %}
+{% load morefilters %}
 
 {% block content %}
     {{ block.super }}
+    {% if form.errors %}
+        <div class="alert alert-danger" role="alert">{% blocktrans %}Please fill out all required fields and select at least one recipient group.{% endblocktrans %}</div>
+    {% endif %}
```

Comment:
error texts are not shown anymore. the respective fields are just marked in red. so i felt some message is needed.